lib/sprig/reap/inputs/environment.rb

```diff
@@ -0,0 +1,24 @@
+module Sprig::Reap::Inputs
+  class Environment
+    attr_reader :input
+
+    def self.default
+      new().env
+    end
+
+    def self.parse(input)
+      new(input).env
+    end
+
+    def initialize(input = nil)
```

Comment:
It'd except `nil` as an argument then, which is what I'm trying to avoid.  With this method signature, passing either no argument (`new()`) or passing in `nil` (`new(nil)`) will fall back to the current `Rails.env`.